classes/layer-generator.bbclass

```diff
@@ -59,6 +59,35 @@ do_create_rootfs_archive() {
     mv ${IMAGE_ROOTFS_TAR} ${LAYER_WORK_DIR}/${DIGEST}
 }
 
+python do_create_whiteouts() {
+    import os
+
+    whiteoutPrefix = ".wh."
+    whiteoutOpaqueDir = whiteoutPrefix + whiteoutPrefix + ".opq"
+
+    whiteouts = d.getVar("LAYER_WHITEOUTS_{}".format(d.getVar("PN")))
+    if whiteouts is None:
```

Comment:
Do you need this check? The following loop will not be executed if whiteouts is None. Right?

Reply:
If we do not add a check in this case, then when we try to split a string,  we will get an error that such an operation cannot be performed for TypeNone. If we don't add this check, then in each layer we will need to write this variable if we don't use whiteouts at all to reduce layer compiler error

Reply:
ok.